plugins/lighthouse/src/components/Audit/index.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { FC, useState, useEffect } from 'react';
+import { useInterval, useBoolean } from 'react-use';
+import { Website, lighthouseApiRef } from '../../api';
+import { useApi } from '@backstage/core';
+import {
+  SparklinesDataByCategory,
+  buildSparklinesDataForItem,
+} from '../../utils';
+import { AuditRow } from './AuditRow';
+export const LIMIT = 10;
+
+export const Audit: FC<{
+  website: Website;
+  categorySparkline: SparklinesDataByCategory;
+}> = ({ website, categorySparkline }) => {
+  const lighthouseApi = useApi(lighthouseApiRef);
+  const fetchWebsite = async (auditId: string) => {
```

Comment:
This one is not necessary as a standalone thing. You may want to make a separate `const runRefresh = () => {}` that does this plus what's inside the useInterval below. Or just move it all inside useInterval. Matter of taste